agent/eni/watcher/watcher_unsupported.go

```diff
@@ -0,0 +1,49 @@
+// +build !linux,!windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package watcher
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+	"github.com/aws/amazon-ecs-agent/agent/statechange"
+)
+
+type UdevWatcher struct {
+	ctx                  context.Context
```

Comment:
Why do you need these for unsupported?

Reply:
These are used in the common platform independent file (watcher_common.go).